source/Symbol/ClangASTContext.cpp

```diff
@@ -1437,8 +1437,10 @@ CompilerType ClangASTContext::CopyType(const CompilerType &src) {
           FileManager file_manager(file_system_options);
           ASTImporter importer(*dst_clang_ast, file_manager, *src_clang_ast,
                                file_manager, false);
-          QualType dst_qual_type(importer.Import(ClangUtil::GetQualType(src)));
-          return CompilerType(this, dst_qual_type.getAsOpaquePtr());
+          if (auto expected = importer.Import(ClangUtil::GetQualType(src))) {
+            QualType dst_qual_type(expected.get());
+            return CompilerType(this, dst_qual_type.getAsOpaquePtr());
+          }
```

Comment:
I believe this will assert in the destructor of llvm::Error when the expected is empty. You'd have to do something like   in CopyDecl: 

```
if (auto expected = importer.Import(ClangUtil::GetQualType(src))) {
  ...
} else { 
  Log *log = lldb_private::GetLogIfAllCategoriesSet(LIBLLDB_LOG_EXPRESSIONS);
  LLDB_LOG_ERROR(log, expected.takeError(), "Couldn't import type: {0}");	
}

```

Reply:
I can add the logging, but calling operator bool() on it should mark it as checked if it's empty

https://llvm.org/doxygen/Support_2Error_8h_source.html#l00235

Reply:
Yup, you are correct. I think the logging here is good, but otherwise it's customary to have an [`llvm::consumeError`](https://llvm.org/doxygen/namespacellvm.html#aee0e70be66536b9ef7b65ae2e5a45959) when you want to explicitly discarding the error.